net/netifyd/files/netifyd.init

```diff
@@ -10,40 +10,84 @@ STOP=50
 USE_PROCD=1
 PROG=/usr/sbin/netifyd
 
+function append_params() {
+	procd_append_param command $@
+}
+
+function append_ifopts() {
+	local filter=0
+	local filter_expr=
+
+	for a in $1; do
+	    case $a in
+	    -F|--device-filter)
+	        filter=1
+	        procd_append_param command $a
+	        ;;
+	    -*)
+	        if [ $filter -gt 0 ]; then
+	            procd_append_param command "${filter_expr#\ }"
+	            filter=0; filter_expr=
+	        fi
+	        procd_append_param command $a
+	        ;;
+	    *)
+	        if [ $filter -gt 0 ]; then
+	            a=${a#\"}; a=${a%\"}: a=${a#\'}; a=${a%\'}
```

Comment:
Rogue colon at the end of the 2nd assignment, should be semi-colon - always something :-)

Reply:
So embarrassed!  Of course I folded that file **after** testing :(

Fixed!  ...and thanks a lot of testing!  Much appreciated.